scripts/build_shapes.py

```diff
@@ -859,6 +858,11 @@ def gadm(
         )
 
     # set index and simplify polygons
```

Comment:
I think you need to replace this comment.

Also, above the code contribtion you could leave a comment e.g.
```
    # renaming 3 letter to 2 letter ISO code before saving GADM file
    # solves issue: https://github.com/pypsa-meets-earth/pypsa-earth/issues/671
    df_gadm["GADM_ID"] = (
        df_gadm["GADM_ID"]
        .str.split(".")
        .apply(lambda id: three_2_two_digits_country(id[0]) + "." + id[1])
    )
 ```